Parser/Token/Token.php

```diff
@@ -69,7 +69,7 @@ public function __construct($name = null, array $options = [])
 
         $this->name     = $name;
         $this->rule     = isset($options['rule']) ? $options['rule'] : new Rule();
-        $this->options &= $options;
+        $this->options = $options;
```

Comment:
It'd be nice if all `=` signs are aligned (as now they are)

```suggestion
        $this->options  = $options;
```

Reply:
Sure will sort that :)